bin/install

```diff
@@ -481,8 +481,8 @@ function sane_environment_check
   end
 
   debug "Verifying Git autocrlf is not enabled"
-  if test (cmd git config --bool core.autocrlf; or echo false) = true
-    abort "Please disable core.autocrlf in your Git configuration."
+  if test (cmd git config core.autocrlf) = false
+    echo "Please disable core.autocrlf in your Git configuration."
```

Comment:
changing this to `echo` means it won't exit on failure, right?

Reply:
@bobthecow You might think so, but when git core.autocrlf was set to "input" it fail and exited the installer. Here is the output from the current installer with core.autocrlf set to input using the verbose and check options.

```fish
$  ./install --verbose --check                                                           http://ohmyfish.dev.topdroplabs.com  [Mon Mar  9 18:48:01]
DEBUG: verbose turned on
Checking for a sane environment...
DEBUG: Command 'awk' is /usr/bin/gawk
DEBUG: Command 'basename' is /usr/bin/basename
DEBUG: Command 'cp' is /bin/cp
DEBUG: Command 'cut' is /usr/bin/cut
DEBUG: Command 'date' is /bin/date
DEBUG: Command 'dirname' is /usr/bin/dirname
DEBUG: Command 'env' is /usr/bin/env
DEBUG: Command 'fish' is /usr/bin/fish
DEBUG: Command 'fold' is /usr/bin/fold
DEBUG: Command 'head' is /usr/bin/head
DEBUG: Command 'mkdir' is /bin/mkdir
DEBUG: Command 'mv' is /bin/mv
DEBUG: Command 'readlink' is /bin/readlink
DEBUG: Command 'rm' is /bin/rm
DEBUG: Command 'sed' is /bin/sed
DEBUG: Command 'sort' is /usr/bin/sort
DEBUG: Command 'tar' is /bin/tar
DEBUG: Command 'tr' is /usr/bin/tr
DEBUG: Command 'which' is /bin/which
DEBUG: Checking for a sane 'head' implementation
DEBUG: Checking for a sane 'sort' implementation
DEBUG: Checking for a working AWK interpreter
DEBUG: Verifying Git implementation is not buggy Git for Windows
DEBUG: Verifying Git autocrlf is not enabled
fatal: bad numeric config value 'input' for 'core.autocrlf' in file /home/docker/.gitconfig: invalid unit
```
I went back and tested the original installer using the three possible values. It looks like I have this wrong. I should be testing for true not false. "false" and "input" are acceptable to the installer but "true" is not.

Reply:
@bobthecow I when back and took another look at the original code. I believe the reason that the value input does not get processed correctly, is the `--bool` option as "input" does not seem to convert to a boolean. I revising the code to:
```fish
if test (cmd git config core.autocrlf; or echo false) = true
    abort "Please disable core.autocrlf in your Git configuration."
```
and testing with the three possible values and it does work as expected with true aborting the installer and false and input being accepted values and the installer continues.

Reply:
@bobthecow I can revise this pull request this way if you prefer it over mine, which I find much simpler.

Reply:
yeah, i like that too, thanks.

Reply:
Sorry, @bobthecow  I am not sure I understand what you mean. Do you like the fix I coded or the idea to remove the `--bool` option?

Reply:
Yes, I mean let's update it to use the code in https://github.com/oh-my-fish/oh-my-fish/pull/742/files#r389878048 

Thanks!

Reply:
Done.